src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/commands.py

```diff
@@ -122,3 +123,18 @@
         c.command('get', 'get_resource')
         c.command('list', 'list')
         c.command('delete', 'delete_resource')
+
+
+# Secret Operations Commands
+secret_operations = create_service_adapter(
+    mgmt_operations_path.format('secret_operations'),
+    'SecretOperations')
+
+with ServiceGroup(__name__, get_devtestlabs_secret_operation,
+                  secret_operations) as s:
+    with s.group('lab secret') as c:
+        c.command('create', 'create_or_update_resource')
+        c.command('show', 'get_resource')
+        c.command('list', 'list')
+        c.command('delete', 'delete_resource')
+        c.generic_update_command('update', 'get_resource', 'create_or_update_resource')
```

Comment:
@tjprescott Seems like there is a confusion. I am not using any custom commands here. It seems like I missed to remove `create_or_update_secret` which i was using it before. Let me know if that is the thing you are referring.